go.mod

```diff
@@ -6,56 +6,36 @@ require (
 	github.com/aws/aws-sdk-go v1.41.10
 	github.com/hashicorp/go-hclog v1.1.0
 	github.com/hashicorp/go-multierror v1.1.1
-	github.com/hashicorp/vault/api v1.4.1
+	github.com/hashicorp/vault/api v1.9.0
```

Comment:
We should probably set the .go-version in this repo to the latest 1.20.x or 1.19.x, since api/v1.9.0's go.mod specifies 1.19: https://github.com/hashicorp/vault/blob/api/v1.9.0/api/go.mod

Reply:
bumped to 1.19